services/web/server/src/simcore_service_webserver/login/_controller/rest/change.py

```diff
@@ -10,12 +10,12 @@
 from servicelib.request_keys import RQT_USERID_KEY
 from simcore_postgres_database.utils_repos import pass_or_acquire_connection
 from simcore_postgres_database.utils_users import UsersRepo
-from simcore_service_webserver.db.plugin import get_asyncpg_engine
 
 from ...._meta import API_VTAG
+from ....db.plugin import get_asyncpg_engine
```

Comment:
question: why is this still bypassing the usual asyncpg engine of sqlalchemy?

Reply:
it is not. This is just a helper function that retrieves it from the app's state

Reply:
it is not. This is just a helper function that retrieves it from the app's state